src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
     // write code here
+    const defaultCompare = (el1, el2) => {
+      const itemOne = `${el1}`;
+      const itemTwo = `${el2}`;
+
+      return itemOne > itemTwo ? +1 : -1;
+    };
+
+    const comparer = arguments.length === 0 ? defaultCompare : compareFunction;
```

Comment:
It's good practice to use 'typeof' to check the type of 'compareFunction' to ensure it is indeed a function before assigning it to 'comparer'.